lexicons/tools/ozone/moderation/defs.json

```diff
@@ -894,6 +896,21 @@
           "description": "The total number of records labeled as a result of the user's reports."
         }
       }
+    },
+    "userAgent": {
+      "type": "object",
+      "description": "User agent information for tracing the source of the action",
+      "required": ["name"],
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name/identifier of the source (e.g., 'automod', 'ozone/workspace')"
+        },
```

Comment:
I assume also using an actual user-agent string (in [this](https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Headers/User-Agent) sense) would also be welcome.